connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -135,7 +134,7 @@ public void stop() {
             WorkerConnector conn = entry.getValue();
             log.warn("Shutting down connector {} uncleanly; herder should have shut down connectors before the" +
                     "Worker is stopped.", conn);
-            conn.stop();
+            conn.shutdown();
```

Comment:
It got confusing inside WorkerConnector since pause() was actually calling stop() on the connector and firing onPaused(), while stop() was calling stop() and firing onShutdown(). That's the only reason.